src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -436,6 +447,35 @@ def to_ase_atoms(self, logger: 'BoundLogger') -> Optional[ase.Atoms]:
 
         return ase_atoms
 
+    def from_ase_atoms(self, ase_atoms: ase.Atoms, logger: 'BoundLogger') -> None:
+        """
+        Parses the information from an ASE Atoms object to the `AtomicCell` section.
+
+        Args:
+            ase_atoms (ase.Atoms): The ASE Atoms object to parse.
+            logger (BoundLogger): The logger to log messages.
+        """
+        # `AtomsState[*].chemical_symbol`
+        chemical_symbols = ase_atoms.get_chemical_symbols()
+        for symbol in chemical_symbols:
+            atom_state = AtomsState(chemical_symbol=symbol)
+            self.atoms_state.append(atom_state)
+
+        # `periodic_boundary_conditions`
+        self.periodic_boundary_conditions = ase_atoms.get_pbc()
+
+        # `lattice_vectors`
+        cell = ase_atoms.get_cell()
+        self.lattice_vectors = ase.geometry.complete_cell(cell) * ureg('angstrom')
```

Comment:
same comment, you can contract into 1 line: the single-use varaible doesn't help the semantics, so it's irrelevant